crates/subspace-runtime/src/lib.rs

```diff
@@ -738,11 +739,20 @@ mod mmr {
     pub type Hash = <Hashing as sp_runtime::traits::Hash>::Output;
 }
 
+pub struct BlockHashProvider;
```

Comment:
You have defined `BlockHashProvider` in various runtimes, but still using `DefaultBlockHashProvider` in practice. Why?

Reply:
Oops! forgot to update this. My bad